game/world/managers/CommandManager.py

```diff
@@ -990,22 +991,45 @@ def worldoff(world_session, args):
 
     @staticmethod
     def serverinfo(world_session, args):
-        os_platform = f'{platform.system()} {platform.release()} ({platform.version()})'
-        message = f'Platform: {os_platform}.\n'
-
-        python_version = f'{platform.python_version()}'
-        message += f'Python Version: {python_version}.\n'
-
-        current_commit_hash = GitUtils.get_current_commit_hash()
-        current_branch = GitUtils.get_current_branch()
-        message += f'Commit: [{current_branch}] {current_commit_hash}.\n'
-
-        server_time = f'{datetime.now()}'
-        message += f'Server Time: {server_time}.\n'
-
-        server_uptime = timedelta(seconds=WorldManager.get_seconds_since_startup())
-        message += f'Uptime: {server_uptime}.'
-
+        short_rev = GitUtils.get_current_commit_hash()[:7]
+        branch = GitUtils.get_current_branch()
+        now = datetime.now()
+        date_str = now.strftime('%Y-%m-%d %H:%M:%S')
+        timezone = time.strftime('%z')
+
+        sys_platform = platform.system()
```

Comment:
I'd simply use the name returned by `platform.system()`, no need to alter it like you are doing below.

Reply:
Also, please make sure to always use single quotes `'` for strings as that's the standard throughout the project.